src/components/Link.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link as ChakraLink, useTheme } from '@chakra-ui/core';
+import { Link as GatsbyLink } from 'gatsby';
+
+function Link(props) {
+  const theme = useTheme();
+  const isCallToAction = props.isCallToAction ? true : false;
+
+  return (
+    <ChakraLink
+      as={GatsbyLink}
+      {...props}
```

Comment:
This needs to go at the end of the component declaration to allow one to override the settings defined below if they need to.

Reasoning:

```jsx
function Example({ variant, ...props }) {
  return (
    <ChakraLink
      {...props} // <- this gets overridden by the next lines
      color={theme.links[variant].color.default}
      {/* Other styles */}
      {...props} // <- this allows the developer using this component to override if they need to
    />
  )
}
```